packages/core/core/src/Transformation.js

```diff
@@ -607,8 +607,20 @@ function normalizeAssets(
 }
 
 function getImpactfulConfigInfo(configs: ConfigMap) {
-  return [...configs].map(([, {resultHash, devDeps}]) => ({
-    resultHash,
-    devDeps: [...devDeps],
-  }));
+  let impactfulConfigInfo = {};
+
+  for (let [configType, {devDeps, resultHash}] of configs) {
+    let devDepsObject = {};
+
+    for (let [moduleName, version] of devDeps) {
+      devDepsObject[moduleName] = version;
+    }
+
+    impactfulConfigInfo[configType] = {
```

Comment:
`configType` is the plugin name here. Could a single transformer run multiple times in a pipeline (meaning something could get overwritten)?

Reply:
We load config at the beginning of loading a pipeline, so even if a transformer were defined twice, the inputs would be the same, so I don't think this is something we need to worry about.